source/common/request_info/request_info_impl.h

```diff
@@ -109,6 +109,8 @@ struct RequestInfoImpl : public RequestInfo {
     last_upstream_rx_byte_received_ = absl::optional<MonotonicTime>{};
   }
 
+  void addBytesReceived(uint64_t bytes_received) override { bytes_received_ += bytes_received; }
```

Comment:
Can we make bytes_received and bytes_sent_ private now and use these accessors everywhere? (Similarly for other accessors added, better to be consistent).

Reply:
Thanks for the comments, all requested changes are in.